src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main (String[] args) {
+        Bulldozer bulldozer = new Bulldozer();
```

Comment:
please use abstract type for variable names
```suggestion
        Machine bulldozer = new Bulldozer();
```